src/Uno.Templates/content/unolib-crossruntime/UnoCrossRuntimeLib/UnoCrossRuntimeLib.Wasm.csproj

```diff
@@ -4,9 +4,10 @@
     <IsPackable>false</IsPackable>
     <RootNamespace>UnoCrossRuntimeLib</RootNamespace>
     <AssemblyName>UnoCrossRuntimeLib</AssemblyName>
+    <UnoRuntimeIdentifier>Wasm</UnoRuntimeIdentifier>
```

Comment:
This is supposed to be added by `Uno.WinUI.Runtime.WebAssembly`

Reply:
Indeed. Actually `Wasm` is wrong. It should be `WebAssembly` which is added properly as you pointed out. My bad!